internal/provider/provider.go

```diff
@@ -80,12 +89,22 @@ func (p *ptProvider) Configure(ctx context.Context, req provider.ConfigureReques
 	if deployURL == "" {
 		resp.Diagnostics.AddError(
 			"Missing Polytomic Deployment URL",
-			fmt.Sprintf("Please set the deployment_url in the provider configuration or the %s environment variable", PolytomicDeploymentURL),
+			fmt.Sprintf(`Please set the "deployment_url" in the provider configuration or the %s environment variable`, PolytomicDeploymentURL),
```

Comment:
I wonder if we should default this to the hosted platform at (gestures vaguely) some point.

Reply:
Agreed! Great idea https://github.com/polytomic/terraform-provider-polytomic/issues/32